windows/web/steamvr.ps1

```diff
@@ -35,13 +35,13 @@ if ($ExternalDriverPaths.Length -eq 0) {
 } else {
     $OpenVrConfig.external_drivers = $ExternalDriverPaths
 }
-[System.IO.File]::WriteAllLines($OpenVrConfigPath, (ConvertTo-Json -InputObject $OpenVrConfig))
+[System.IO.File]::WriteAllLines($OpenVrConfigPath, (ConvertTo-Json -InputObject $OpenVrConfig -Compress))
```

Comment:
Why do we need `-Compress` again?

Reply:
Some strange PowerShell bug, without that it throws a JSON format error
```
+ [System.IO.File]::WriteAllLines($OpenVrConfigPath, (ConvertTo-Json 
-InputObject  ...
+                                                     
~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (@{config=System...t[]; versio 
   n=1}:PSObject) [ConvertTo-Json], InvalidOperationException
    + FullyQualifiedErrorId : JsonStringInBadFormat,Microsoft.PowerShell.Comma 
   nds.ConvertToJsonCommand
```
If I remember correctly, I saw something mention it had to do with trailing backslashes in strings causing that, and the file usually has a path or two with trailing backslashes.